app/components/g-maps-container/component.js

```diff
@@ -10,22 +10,8 @@ export default Ember.Component.extend({
   zoom: 4,
   markers: computed('model.[]', function() {
```

Comment:
I'm pretty sure you can just use [`computed.map`](http://emberjs.com/api/classes/Ember.computed.html#method_map)